projects/OdroidXU3/options

```diff
@@ -88,10 +88,10 @@
   # Xorg Graphic drivers to use (all / i915,i965,r200,r300,r600,nvidia,nouveau)
   # Space separated list is supported,
   # e.g. GRAPHIC_DRIVERS="i915 i965 r300 r600 radeonsi nvidia nouveau"
-    GRAPHIC_DRIVERS=""
+    GRAPHIC_DRIVERS="lima"
```

Comment:
Why you set this?

Reply:
mesa failed with: Problem encountered: EGL requires dri.
So this needs some galium driver.

Reply:
https://github.com/libretro/Lakka-LibreELEC/blob/master/config/graphic#L67L70
https://github.com/mesa3d/mesa/blob/master/meson.build#L219L248

Reply:
Is this wrong?

Reply:
XU4 needs panfrost as is a midgard chip

Reply:
mesa removed.